src/RawPOSIX/src/safeposix/dispatcher.rs

```diff
@@ -1320,6 +1323,7 @@ pub fn lindrustinit(verbosity: isize) {
         thread_table: interface::RustHashMap::new(),
         signalhandler: interface::RustHashMap::new(),
         sigset: interface::RustHashMap::new(),
+        pendingsigset: interface::RustHashMap::new(),
```

Comment:
are you working off an old branch? these shouldnt be reintroduced

Reply:
I took a pull from the vmmap-alice branch from the standalone repo for the RawPosix. Sorry, I will comment out all the places where this is being used.

Reply:
it was removed in a recent PR here so it needs to still be removed, not commented

Reply:
> I took a pull from the vmmap-alice branch from the standalone repo for the RawPosix. Sorry, I will comment out all the places where this is being used.

Just a suggestion: my perferred way when I migrate a PR from old RawPOSIX to lind-wasm mono-repo is that:
1. create a new branch in lind-wasm
2. copy and replace the entire RawPOSIX folder from old RawPOSIX and delete the .git folder
3. commit the changes
4. do a rebase and move this new commit right after mono-repo's commit
5. Since the commit history is changed, doing a PR on the branch would cause github reports a lot of things. One extra step to avoid this is just
- 1. create an another new branch forked from main
- 2. replace the RawPOSIX folder from the previous rebased branch to the new branch and do PR on this branch. This should allow the branch contains the most latest information while holding your change

Current flow, this may cause VMMAP changes overwrite existing things if not carefully viewed:
1. (old RawPOSIX repo) old RawPOSIX repo's main -> VMMAP commits
2. (mono-repo) old RawPOSIX repo's main -> mono-repo start commit -> a bunch of other commits -> migrated VMMAP changes

Rebased Approach, this should ensure that the VMMAP changes will not overwrite any new commits after mono-repo is created:
1. (old RawPOSIX repo) old RawPOSIX repo's main -> VMMAP commits
2. (mono-repo) old RawPOSIX repo's main -> mono-repo start commit -> migrated VMMAP changes -> a bunch of other commits